Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -733,7 +733,7 @@ ExceptionOr<void> GPUQueue::copyExternalImageToTexture(ScriptExecutionContext& c
         if (!supportedFormat || !(destinationTexture->usage() & GPUTextureUsage::RENDER_ATTACHMENT))
             copyDestination.mipLevel = INT_MAX;
 
-        m_backing->writeTexture(copyDestination, newImageBytes ? newImageBytes : imageBytes.data(), sizeInBytes, dataLayout.convertToBacking(), convertToBacking(copySize));
+        m_backing->writeTexture(copyDestination, newImageBytes ? std::span { newImageBytes, sizeInBytes } : imageBytes, dataLayout.convertToBacking(), convertToBacking(copySize));
```

Comment:
This doesn't look right, mixing span and pointer in the same expression. Surprised it compiles.

Reply:
`imageBytes` is a span. I don't think I am mixing span and pointer?

Reply:
Oops, not sure why I misunderstood.